app/controllers/restaurant_controller.rb

```diff
@@ -0,0 +1,11 @@
+class RestaurantController < ApplicationController
+
+  def show
+    service = RevItUpService.new(current_user.lat, current_user.lng, meters_to_miles, params["price"])
+    @restaurant = RevItUpFacade.new(service.restaurant)
+  end
+
+  def meters_to_miles
+    (params["distance"][0].to_i * 1609).to_i
```

Comment:
Converter to switch the API's use of meters to miles for common use in the states.